src/mon/MDSMonitor.cc

```diff
@@ -1557,6 +1557,13 @@ bool MDSMonitor::has_health_warnings(vector<mds_metric_t> warnings)
   return false;
 }
 
+bool MDSMonitor::has_any_health_warning()
+{
+  return std::any_of(
+    pending_daemon_health.begin(), pending_daemon_health.end(),
+    [](auto& it) { return it.second.metrics.size() > 0 ? true : false; });
```

Comment:
OK. This is new (I think). Wasn't this just `!it.second.metrics.empty()`?

Reply:
Nope. It's exactly same as before.

Reply:
kk, then that was a different change then. Anyway, please use `!it.second.metrics.empty()` and run this through fs suite 👍

Reply:
Okay. Done.